data/collector_test.go

```diff
@@ -14,27 +14,12 @@ import (
 	"go.uber.org/zap/zapcore"
 	v1 "go.viam.com/api/app/datasync/v1"
 	"go.viam.com/test"
-	"go.viam.com/utils/protoutils"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
-	"google.golang.org/protobuf/types/known/structpb"
 
 	"go.viam.com/rdk/logging"
-	"go.viam.com/rdk/services/datamanager/datacapture"
 )
 
-type structReading struct {
```

Comment:
this type was already defined in the data package... which is further evidence that data and datamanager/datacapture should have always been one package.